android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -130,8 +130,6 @@ class QuestionnaireActivity : BaseMultiLanguageActivity() {
           }
 
           registerFragmentResultListener()
-
-          viewModel.setProgressState(QuestionnaireProgressState.QuestionnaireLaunch(false))
```

Comment:
why is this no longer necessary?

Reply:
The progress dialog is dismissed in the `finally` section of the `try-catch` block.